pkg/server/server.go

```diff
@@ -378,29 +378,31 @@ func (s *Server) SendVerificationLink(c *gin.Context, email string) {
 }
 
 // VerifyEmail marks a user email as verified
-func (s *Server) VerifyEmail(c *gin.Context) {
+func (s *Server) VerifyEmail(c *gin.Context, redirectURL string) {
 	jwtUser, exists := c.Get(types.JWTUserCredentialsKey)
 	if !exists {
 		s.redirectError(c, errors.ErrInvalidAccessToken)
+		c.Redirect(http.StatusUnauthorized, redirectURL)
```

Comment:
Can you explain this change as code comments?

Reply:
Trying to understand - why not redirect to error screen? Or am I missing something?

Reply:
I think its fine.

Reply:
Okay. My thought process was to redirect the user to the same link and let UI take care if any error is there as UI will better know whether to redirect him to an error page (since we have quite a few possibility of errors ) or to the login page(if in case the token is not there).